packages/ckeditor5-dev-release-tools/lib/utils/publishpackageonnpmcallback.js

```diff
@@ -20,12 +20,22 @@ module.exports = async function publishPackageOnNpmCallback( packagePath, taskOp
 	const upath = require( 'upath' );
 	const fs = require( 'fs-extra' );
 
-	await tools.shExec( `npm publish --access=public --tag ${ taskOptions.npmTag }`, { cwd: packagePath, async: true, verbosity: 'error' } )
-		.catch( () => {
+	const result = await tools.shExec( `npm publish --access=public --tag ${ taskOptions.npmTag }`, {
+		cwd: packagePath,
+		async: true,
+		verbosity: 'error'
+	} )
+		.catch( e => {
 			const packageName = upath.basename( packagePath );
 
+			if ( e.message.includes( 'code E409' ) ) {
```

Comment:
I'm not sure if `e` is always an Error instance.
```suggestion
			if ( e.toString().includes( 'code E409' ) ) {
```